packages/wonder-blocks-toolbar/src/components/toolbar.tsx

```diff
@@ -40,22 +40,16 @@ type Props = AriaProps & {
      */
     subtitle?: string;
     /**
-     * The main title rendered in larger bold text.
+     * The main title rendered in larger bold text. It also supports rendering
+     * React nodes (use with caution).
      */
-    title?: string;
+    title?: string | React.ReactNode;
```

Comment:
This is the main functional change right?

Also, I believe that `ReactNode` contains `string` so this can probably be simplified to `title?: React.ReactNode`.

Reply:
That's a good point, although, I'd prefer to keep it that way as Storybook autogenerates docs based on this types. By keeping this, we signal devs that "string" is the default way to go, but that they can still rely on `ReactNode` as an alternative approach.